tests/test_model.py

```diff
@@ -76,12 +75,9 @@ def test_cpes():
     assert ps2node
     ps3node = ProductNode.objects.create(parent=pv2node, obj=ps3)
     assert ps3node
+    ProductNode.objects.create(parent=ps1node, obj=pvariant1)
 
-    assert pv1.cpes == ("cpe:/o:redhat:enterprise_linux:8",)
-    assert sorted(p1.cpes) == [
-        "cpe:/o:redhat:Brantabernicla:8",
-        "cpe:/o:redhat:enterprise_linux:8",
-    ]
+    assert p1.cpes == ("cpe:/o:redhat:Brantabernicla:8",)
```

Comment:
I think the old asserts were mostly correct, but the new variant needs to be linked into the taxonomy correctly. Then we can check that Variant CPEs are discovered correctly for a product with two different versions that each have their own variant:
```suggestion
    # Version 1 for RHEL 7 has 1 child variant, so 1 CPE is reported
    assert pv1.cpes == ("cpe:/o:redhat:enterprise_linux:7")
    # Product 1 for RHEL has 2 child variants, one for each RHEL version
    # So 1 CPE for RHEL 7 and 1 CPE for RHEL 8 is reported
    assert sorted(p1.cpes) == [
        "cpe:/o:redhat:Brantabernicla:8",
        "cpe:/o:redhat:enterprise_linux:7",
    ]
```